common/opinion_modifiers/hfe_opinion.txt

```diff
@@ -133,31 +143,498 @@ hfe_opinion_future_vs_neighbor = {
 	}
 }
 
-# hate slavers&purgers
+# hate slavers & purgers
 hfe_opinion_future_vs_bad = {
 	trigger = {
-		has_country_flag = hfe_future
-		from = {
-			or = {
+		OR = {
+			has_country_flag = hfe_future
+			AND = {
+				NOT = { has_global_flag = hfe_features_opinion_disable }
+				is_fallen_empire_xenophile = yes
```

Comment:
Not intended for vanilla

Reply:
It is now.

That's why it is tied to your Global Flag about Opinions.